schema/defs-windows.json

```diff
@@ -0,0 +1,20 @@
+{
+    "definitions": {
+        "cpuShares": {
+            "description": "Relative weight to other containers with CPU Shares defined",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 10000
+        },
+        "cpuSharesPointer": {
+            "oneOf": [
+                {
+                    "$ref": "#/definitions/CPUShares"
```

Comment:
I don't have much of a preference between `cpuShares` and `CPUShares`, but we should pick one and stick to it ;).